cmd/nodepool/create.go

```diff
@@ -32,6 +36,9 @@ type CreateNodePoolOptions struct {
 	RootVolumeType  string
 	RootVolumeIOPS  int64
 	RootVolumeSize  int64
+	KubevirtMemory  string
```

Comment:
This needs to be decomposed into platforms same way we did for `create cluster`

Reply:
I agree
Do you think I can take this task after merging this PR?

Reply:
A follow up PR seems reasonable so we can keep PR size controlled. Please include a // TODO for this here.

Reply:
Done
I will start this change after this PR merge